pom.xml

```diff
@@ -123,11 +123,45 @@
           <artifactId>maven-bundle-plugin</artifactId>
           <version>3.3.0</version>
           <extensions>true</extensions>
+          <dependencies>
+            <dependency>
+              <!-- Required to make the plugin work on JDK9. See https://issues.apache.org/jira/browse/FELIX-5698. -->
```

Comment:
Good to have this comment.
Can we improve the comment to clarify when this workaround becomes obsolete (without having to follow the link)?

Proposal comment: "TODO Remove when upgrading to maven-bundle-plugin 3.4.0. This is a workaround for https://issues.apache.org/jira/browse/FELIX-5698"

Reply:
The linked JIRA is not yet resolved, so at this point I don't know which plugin version will receive the fix. I could change the comment to something like: `TODO: remove when upgrading maven-bungle-plugin which includes fix for https://issues.apache.org/jira/browse/FELIX-5698`